src/main/scala/firrtl/annotations/AnnotationUtils.scala

```diff
@@ -3,9 +3,15 @@
 package firrtl
 package annotations
 
+import net.jcazevedo.moultingyaml._
+import firrtl.annotations.AnnotationYamlProtocol._
+
 import firrtl.ir._
 
 object AnnotationUtils {
+  def toYaml(a: Annotation): String = a.toYaml.prettyPrint
+  def fromYaml(s: String): Annotation = s.parseYaml.convertTo[Annotation]
```

Comment:
Hm I see. I made a fake transform that didn't do anything for some global params I used across multiple transforms.